src/test/e2e/api/client/feature.optimal304.e2e.test.ts

```diff
@@ -139,6 +140,14 @@ describe.each([
             TEST_AUDIT_USER,
         );
 
+        await app.services.eventService.storeEvent({
```

Comment:
This is added as the last event to prevent the IDs of the existing events above this one from shifting.